### PR TITLE
ssd: added documentation on enabling trim

### PR DESCRIPTION
**Pull request by anjandev**
My old PR had a commit on master. On my local machine, I changed the branch cause I needed to branch off master for other things. However, github doesnt let me change my branch in the PR :angry: .

This PR supersedes  #164. I implemented @flexibeast's changes.

@ericonr

### CODE REVIEW

File: src/config/ssd.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+If the DISC-GRAN (discard granularity) and DISC-MAX (discard maximum bytes)
+columns are non-zero, that means the block device has TRIM support. If your
+solid state drive partition does not show TRIM support, please verify that you
+chose a file system with TRIM support (ext4, btrfs, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. Note that to run
+`fstrim` on F2FS file systems, you need to be running kernel 4.19 or greater.
+For example, if your / directory is on an SSD:
```
Comment (ericonr) [inline src/config/ssd.md:18]: I'd prefer if you mentioned F2FS in the previous paragraph.

"(ext4, Btrfs, F2FS, etc.). Note that F2FS requires kernel 4.19 or above to support TRIM."

**Comment by anjandev** [comment]
Done.

File: src/config/ssd.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+If the DISC-GRAN (discard granularity) and DISC-MAX (discard maximum bytes)
+columns are non-zero, that means the block device has TRIM support. If your
+solid state drive partition does not show TRIM support, please verify that you
+chose a file system with TRIM support (ext4, Btrfs, F2FS, etc.).
+
+To run TRIM one-shot, you can run
+[`fstrim(8)`](https://man.voidlinux.org/fstrim.8) manually. Note that F2FS
+requires kernel 4.19 or above to support TRIM. For example, if your / directory
+is on an SSD:
```
Comment (ericonr) [inline src/config/ssd.md:18]: The F2FS thing is still a bit out of place. Why not put it together with the list of file systems that support TRIM?

Comment (ericonr) [reply src/config/ssd.md:18]: > (ext4, Btrfs, F2FS, etc.). Note that F2FS requires kernel 4.19 or above to support TRIM.

**Pull request merged**